src/components/graph/graph.ts

```diff
@@ -513,15 +508,14 @@ export default class GraphComponent {
       .call(
         this._zoomHandler.transform,
         d3.zoomIdentity
-          .translate(this._homeLocation.x, this._homeLocation.y)
+          .translate(this._homeLocation.translation.x, this._homeLocation.translation.y)
           .scale(this._homeLocation.scale),
       )
   }
 
-  private _setHomeLocation(location: IHomeLocation): void {
+  private _setHomeLocation(location: ITransform): void {
```

Comment:
This function isn't actually expecting a `ITransform`, it needs a `IHomeLocation` type. This signature is quite misleading.